end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -567,10 +608,12 @@ head(chromPeaks(Met_test))
 ```{r echo=FALSE}
 #' Plot test chromatogram
 par(mfrow = c(1, 2))
-plot(cystine_test, main = "Cystine", col = paste0(col_phenotype[bpc$phenotype], 80))
+plot(cystine_test, main = "Cystine", col = paste0(col_sample, 80),
```

Comment:
just coloring also the peaks with a background color matching the sample color. It's a little cumbersome to do that, but it looks nicer.